src/tests/ftest/ior/intercept_multi_client.py

```diff
@@ -56,8 +56,9 @@ def test_ior_intercept_multi_client(self):
         # Verifying write performance
         self.assertTrue(float(with_intercept[0][max_mib]) >
                         write_x * float(without_intercept[0][max_mib]))
-        self.assertTrue(float(with_intercept[0][min_mib]) >
-                        write_x * float(without_intercept[0][min_mib]))
+        # DAOS-5857, DAOS-9237 Do not check min performance.
+        #self.assertTrue(float(with_intercept[0][min_mib]) >
+        #                write_x * float(without_intercept[0][min_mib]))
```

Comment:
Should we actually just remove this code, and leave a corresponding comment about checking just max and mean?

Reply:
Yes!  Previously we run for 1 iteration and check min/mean/max which doesn't make a lot of sense, with this PR and https://github.com/daos-stack/daos/commit/52841e03e646e64ba96a00de68912ae05ba22dd9 an important part of the change is to increase the repetitions to 3, then when we check mean and max we're effectively giving occasional slow runs a free pass, but if the whole thing is bad then it'll still fail.